contracts/src/utils/cryptography/ecdsa.rs

```diff
@@ -8,7 +8,6 @@ use alloy_primitives::{address, uint, Address, B256, U256};
 use alloy_sol_types::{sol, SolType};
 use stylus_sdk::{
     call::{self, Call, MethodError},
-    storage::TopLevelStorage,
```

Comment:
Let's try removing all occurrences of `TopLevelStorage` from the whole project, theoretically we shouldn't need it now

Reply:
Yeah. Correct. We need implement them with these changes

Reply:
Thanks for the review @0xNeshi @qalisander . Removed all occurrences of `TopLevelStorage`